openedx/core/djangoapps/organization_api/api/views.py

```diff
@@ -0,0 +1,39 @@
+"""
+Organizations API views.
+"""
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from util.organizations_helpers import get_organization_by_short_name
+
+
+class OrganizationsView(APIView):
```

Comment:
If you dig through the many, unnecessary, layers of utility and data functions you will eventually arrive at the `Organization` model. I'm not asking you to get rid of all of those unnecessary layers...yet, since you didn't write them. I am asking you to start the process of ending this unnecessary code.

We are not fully taking advantage of DRF, and that is a shame because we end up writing/testing a bunch of debt. Ditch the debt, and use a `ReadOnlyModelViewSet`. Your code will be cleaner. Your API will be able to return a single, or all, organizations. We will use DRF serializers, instead of this weird abstraction layer.